packages/components/MenuButton/src/chevron.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="11" height="6" viewBox="0 0 11 6" style="margin-top: 6px;">
+  <path fill="currentColor" d="M0.646447 0.646447C0.841709 0.451184 1.15829 0.451184 1.35355 0.646447L5.5 4.79289L9.64645 0.646447C9.84171 0.451185 10.1583 0.451185 10.3536 0.646447C10.5488 0.841709 10.5488 1.15829 10.3536 1.35355L5.85355 5.85355C5.65829 6.04882 5.34171 6.04882 5.14645 5.85355L0.646447 1.35355C0.451184 1.15829 0.451184 0.841709 0.646447 0.646447Z" />
+</svg>
```

Comment:
Were we able to solve the SVG bundling issue downstream? I thought we couldn't include non-inline SVG's like this in our components? @samuelfreiberg is the last person I remember who dealt with that.

Reply:
I added type definition file and it worked. But not for snapshot tests.

Reply:
I think the issue is not with FluentTester / our snapshot tests, but when we try to consume to code downstream. We don't bundle the assets in sdx platform I think, so if a client tries to use MenuButton inside their SDX it will fail. I am not _quite_ sure why, but I remember Sammy backed out changes that included SVG's for checkbox

Reply:
Each SDX ownder would beed to add support for resolving SVGs in their bundle, packaging them into their NuGets/appx's, and building them into Office setup delivery so that they're available to clients on C2R installs.

If clients don't do this, they get bundling errors after syncing to the latest FURN version. 

I'm not sure if we ever came to a conclusion on this issue, adding @FalseLobster Did we ever have a discussion about this?

Reply:
Should it be a problem for the MenuButton? Because I added type definition inside the component. Can I check it somewhere if it would work for our clients?

Reply:
You could also just keep the svg inline and avoid the problem altogether. That's what we've done on the apple side for ActivityIndicator / Shimmer

Reply:
@samuelfreiberg  chevron.svg ?

Reply:
@Saadnajmi , if I add SVG inline, there will be problems with colors again

Reply:
@ValentinaKozlova just wondering, what are those color issues?  Do we need guidance on SVG usage for this issue if others may run into it? (not sure what Warren's github username is to mention him here)

I'm intrigued that inline text vs the svg file give different colors, should we consider those bugs on rn-svg?

Reply:
I'll double check, because I fixed one of the issues. It was not possible to pass custom color. I'll check, maybe it's working now for inline SVGs also

Reply:
We pass a custom color to the ActivityIndicator inline SVG as so: https://github.com/microsoft/fluentui-react-native/blob/eae3b80724f357040f5a6d724bdf0fe14ac6d5f5/packages/experimental/ActivityIndicator/src/ActivityIndicator.tsx#L17

It may not be the same for you, but there's an example if it helps